lib/matplotlib/path.py

```diff
@@ -341,6 +342,16 @@ def make_compound_path(cls, *args):
                 codes[i:i + len(path.codes)] = path.codes
             i += len(path.vertices)
 
+        # remove internal STOP's, replace kinal stop if present
```

Comment:
typo: final

Reply:
I would also not bother with restoring the final STOP given that it doesn't matter and we may deprecate STOPs.
And even though this is clearly a bugfix, it also changes the behavior of the function wrt STOP codes, so it may change it a tiny bit more (in a less observable fashion! the final STOP doesn't matter, unlike the middle ones) to keep the implementation simpler.

Reply:
If that is acceptable, then I will document it and go with ignoring all STOPs.

Reply:
at least for me, it seems ok.